src/modules/privacy-idea-administration/token-verify.params.ts

```diff
@@ -0,0 +1,14 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsString, IsNotEmpty } from 'class-validator';
+
+export class TokenVerifyBodyParams {
+    @IsString()
+    @IsNotEmpty()
+    @ApiProperty({ type: String })
+    public readonly personId!: string;
```

Comment:
`personId` is our internal DB ID? Could use "IsUUID()" Decorator instead of string and notEmtpy